src/angularjs/src/app/users/detail/users-detail.html

```diff
@@ -45,7 +45,7 @@ <h1 ng-if="user.editing">Edit User</h1>
             <div class="form-group">
```

Comment:
We do want to show it, disabled, so the role is visible but not editable, so probably shouldn't hide it with an `ng-if`. Considering it's necessary to be an admin or org admin to get to the user management page at all, I'll just go ahead and remove the first two tests.